tests/ExamplesTest.php

```diff
@@ -360,6 +360,10 @@ public function testQueryableEncryptionExamples(string $file, string $expectedOu
 
         $this->skipIfServerVersion('<', '7.0.0', 'Queryable encryption tests require MongoDB 7.0 or later');
 
+        if (! $this->isReplicaSet()) {
```

Comment:
```suggestion
        if ($this->isStandalone()) {
```

This should permit sharded clusters, as is done in: https://github.com/mongodb/mongo-php-library/blob/d4cdf057a67cb99a32db8984a16959bfa7ca7eb5/tests/DocumentationExamplesTest.php#L1808-L1810

While writing this, I also realized that `isShardedClusterUsingReplicasets()` is redundant with MongoDB 3.6+. Opened [PHPLIB-1202](https://jira.mongodb.org/browse/PHPLIB-1202) to track that.

Reply:
Ah, yes. I used the negative checks since examples are never tested on sharded clusters, but nonetheless it makes more sense to explicitly skip on stand-alone in case we end up testing on sharded clusters after all.